packages/test/test-service-load/package.json

```diff
@@ -37,6 +37,7 @@
 		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
 		"start": "node ./dist/nodeStressTest.js",
 		"start:frs": "node ./dist/nodeStressTest.js --driver routerlicious --driverEndpoint frs --profile ci_frs",
+		"start:frs:canary": "node ./dist/nodeStressTest.js --driver routerlicious --driverEndpoint frsCanary --profile ci_frs",
```

Comment:
I really love your change overall, but I think this is wrong way to encode it (and we have same problem with ODSP).
The endpoint is FRS. What we deal here is ring. And there could be more rings in the future. For example, we might want to stand up "AFR in a box" dev-environment (as my team is getting to start making changes and might want to have early feedback here).

It's very useful to be able to see overall picture for a given service in telemetry, and it's somewhat weird that we have to do Data_driverEndPoint contains "frs" for that - I think we should have clean separation between type of service and rings.

Reply:
By rings you mean frs, frs-canary, odsp, odsp-df and your service would be frs and odsp?

We can have a new option that differentiates the service and ring and I'm assuming the command would look something like: 
`"start:frs:canary": "node ./dist/nodeStressTest.js --driver routerlicious --service frs --ring canary --profile ci_frs",`

Reply:
Yes, something like that.

Every service and every application have deployment rings. Canary / PROD are the ring for AFR, so is "SPDF / MSIT / PROD" for ODSP, or DF / PROD for Loop
Basically, same builds are showing up in rings at different times, but it's the same service, same endpoint (in a sense that API and behavior is exactly the same).

Reply:
service might be redundant with driverEndpoint